bindings/protocols/bacnet/index.html

```diff
@@ -1,1192 +1,1751 @@
+                        "default": 0
+                      },
+                      "toState": {
+                        "type": "string",
+                        "enum": [
+                          "fault",
+                          "offnormal",
+                          "normal"
+                        ]
+                      },
+                      "eventType": {
+                        "type": "string",
+                        "enum": ["regular", "ack", "syncrequired"],
+                        "default": "regular"
+                      },
+                      "origSource": {
+                        "type": "string",
+                        "format": "iri-reference"
+                      },
+                      "values": {
+                        "type": "object"
+                      },
+                      "inactive": {
+                        "type": "integer",
+                        "enum": [0, 1],
+                        "default": 0
+                      }
+                    },
+                    "required": [
+                      "source",
+                      "time",
+                      "prio",
+                      "toState"
+                    ]
+                },
+                "forms": [
+                    {
+                        "href": "bacnet://2/15,2/102",
+                        "op": [
+                            "subscribeevent",
+                            "unsubscribeevent"
+                        ]
+                    }
+                ],
+              },
+            }
+        }
+        </pre>
+
+        <p>
+            The resulting example payloads are available below.
+        </p>
+
+        <pre id="example-event-data" class="example" title="Example event data">
+            {
+                "source": "bacnet://5/0,20",
+                "time": "2024-07-30T12:33:00+00:00",
+                "prio": 150,
+                "message": "Equipment overheat",
+                "ackReq": 1,
+                "toState": "offnormal",
+                "values": {
+                    "deadband": 0.5,
+                    "exceedLim": 50,
+                    "exceedValue": 55,
+                    "statusFlags": [0, 0, 0, 0]
+                }
+            }
+        </pre>
+
+        <p>
+            [[[#example-event-data]]] shows an event notification because of an overheating equipment. Event <code>message</code>
+            gives this as a useful hint, <code>prio</code> is set accordingly. <code>values</code> object gives additional details
+            about the event that the current value is 55 and the exceeded limit was 50. The unit of these values can be found at the
+            <code>source</code> object that triggered this event.
+            Because overheating events are important for the operator to notice, <code>ackReq</code> is set, so the operator definitely
+            will see the event, even after the overheating condition is over.

```

Comment:
```suggestion
            [[[#example-event-data]]] shows an event notification because of an overheating piece of equipment. Event <code>message</code>
            gives this as a useful hint; <code>prio</code> is set accordingly. <code>values</code> object gives additional details
            about the event — that the current value is 55, and the exceeded limit was 50. The unit of these values can be found at the
            <code>source</code> object that triggered this event.
            Because overheating events are important for the operator to notice, <code>ackReq</code> is set, so the operator definitely
            will see the event, even after the overheating condition is over.
```